### PR TITLE
v4.0.0 Release

### PR DESCRIPTION
**Pull request by TiCampbell**
Includes the following updates:

- Application renewal and email notifications
    - Note that the notification buffer (to notify each user one month before their renewal date) was removed
- Updates to container to allow Django server and qcluster processes
- Genericization of renewal fields to use for initial applications as well
- Added ETL from v3.0 to v4.0.0
- Switched logging to external database

Changes have been fully tested in `dev` and on the Get FoCo test site.

### CODE REVIEW

File: getyour/logger/tests.py
Comment (penoptech) [inline getyour/logger/tests.py:1]: Do you have strong feelings about deleting files like these when they're not used?

File: getyour/logger/views.py
Comment (penoptech) [inline getyour/logger/views.py:1]: Do you have strong feelings about deleting files like these when they're not used?

**Review by penoptech — APPROVED** [review]
@TiCampbell - All of the corrections I would make are superficial and won't prevent us from moving forward with deploying v4.

File: getyour/app/backend.py [hunk 1]
```diff
@@ -762,3 +795,175 @@ def what_page_renewal(last_renewal_action):
 
     # Default return if all pages are present
     return None
+
+
+def check_if_user_needs_to_renew(user_id):
+    """Checks if the user needs to renew their application
+    Args:
+        user (User): The user object
```
[the remark this answers is not in the document]
Comment (TiCampbell) [reply getyour/app/backend.py]: Good catch. Done!

File: getyour/app/backend.py [hunk 2]
```diff
@@ -37,12 +38,13 @@
 from django.db.models.fields.files import FieldFile
 from django.core.serializers.json import DjangoJSONEncoder
 from phonenumber_field.phonenumber import PhoneNumber
-from app.models import HouseholdMembers, EligibilityProgram, IQProgramRD, IQProgram
-from log.wrappers import LoggerWrapper
+from app.models import HouseholdMembers, EligibilityProgram, IQProgramRD, IQProgram, User, Household, AddressRD
+from app.constants import notification_buffer_month
```
[the remark this answers is not in the document]
Comment (TiCampbell) [reply getyour/app/backend.py]: Another good catch. I need to pay more attention to the pylint messages - I also broke the too-long import line above this into multiple lines.

File: getyour/logger/tests.py
Comment (TiCampbell) [reply getyour/logger/tests.py:1]: I don't. Removed!

File: getyour/logger/views.py
Comment (TiCampbell) [reply getyour/logger/views.py:1]: I don't. Removed!

**Pull request merged**